servers/xgboostserver/xgboostserver/requirements.txt

```diff
@@ -1,3 +1,3 @@
-scikit-learn == 0.20.3
-numpy >= 1.8.2
+scikit-learn == 0.23.0
```

Comment:
Similar as above - do you know why the downgrade was required?

Reply:
@adriangonz same sorry about this wrong push. I have fixed the sklearn, numpy, xgb versions.